### PR TITLE
6 article page

### PR DESCRIPTION
**Pull request by AlexMSm**

### CODE REVIEW

**Comment by AlexMSm** [comment]
Just saw I jumped the gun on task 8 with the comment box, apologies

File: src/Components/Article.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  useEffect(() => {
+    setArticleLoading(true);
+    getArticleById(article_id).then((res) => {
+      setArticle(res);
+      setArticleLoading(false);
+    });
+  }, [article_id]);
+
+  if (articleLoading) return <h3>Article loading...</h3>;
```
Comment (jimstev2447) [inline src/Components/Article.jsx:22]: this is a good opportunity start thinking about error handling :)

File: src/Components/ArticleComponents/CommentCard.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+export default function CommentCard({
+  author,
+  body,
+  comment_id,
+  formatedDate,
+  votes,
+}) {
+  const voteCounterColour = (count) => {
```
Comment (jimstev2447) [inline src/Components/ArticleComponents/CommentCard.jsx:8]: This is for ticket 7

File: src/Components/ArticleComponents/CommentContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import CommentCard from "./CommentCard";
+import { useState, useEffect } from "react";
+import { getCommentsByArticleById } from "../../api";
+
+export default function CommentsContainer({ article_id }) {
+  const [comments, setComments] = useState([]);
+  const [commentsLoading, setCommentsLoading] = useState(true);
+
+  useEffect(() => {
+    setCommentsLoading(true);
```
Comment (jimstev2447) [inline src/Components/ArticleComponents/CommentContainer.jsx:10]: This is for ticket 8

File: src/Components/ArticleComponents/CommentContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    getCommentsByArticleById(article_id).then((res) => {
+      res.sort((b, a) => a.votes - b.votes);
```
Comment (jimstev2447) [inline src/Components/ArticleComponents/CommentContainer.jsx:12]: why are you sorting by votes here? do you have the ability to query this on your be?

**Pull request merged**